src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -152,16 +146,20 @@ public JSONArray filterQueryParam(String request) {
         return processAggregatedObject.processQueryAggregatedObject(criteriasJsonNode.toString(), databaseName, aggregationCollectionName);
     }
 
-    /**
-     * This method takes takes the text as input and replaces all the instances
-     * of the searchQueryPrefix (defined in application.properties) with the aggregated
-     * object name (also defined in application.properties) and return the edited text.
-     *
-     * @param  txtObject
-     *     A JSONObject with text to be replaced
-     * @return String text with search prefix replaced by the name for aggregated object
-     */
-    private String editObjectNameInQueryParam(JSONObject txtObject) {
-        return Pattern.compile("("+ searchQueryPrefix + ".)").matcher(txtObject.toString()).replaceAll(objectName +".");
-    }
+    // /**
```

Comment:
Also here, this whole block is commented out.

Reply:
removed